### PR TITLE
Roll 16 edit page

### PR DESCRIPTION
**Pull request by brickgriff**
https://github.com/bgriffiniv/rollme/issues/21

### CODE REVIEW

**Comment by brickgriff** [comment]
Try running this with `> ionic serve -cls` and check out the edit function. The data service function is the next feature. That will persist user data and simulate firebase connections/calls.

File: src/app/profile/profile.page.ts [hunk 1]
```diff
@@ -1,23 +1,47 @@
...
+  constructor(private route: ActivatedRoute, private router: Router) {
+    console.log("Profile page started (constructor)");
+    var defaultUser = {
       name : "Burnest Griffin IV",
       company : "IdeaLogic",
       role : "Developer"
     };
 
-    console.log("Profile page started");
+    this.route.data.subscribe((data) => {
```
Comment (brickgriff) [inline src/app/profile/profile.page.ts:21]: This promise only jogs things along after a submission. It doesn't matter if we watch `data`, `params`, or `queryParams`.

File: src/app/list/list.page.ts [hunk 2]
```diff
@@ -28,10 +28,15 @@ export class ListPage implements OnInit {
         icon: this.icons[Math.floor(Math.random() * this.icons.length)]
```
Comment (brandlxee11) [inline src/app/list/list.page.ts]: What is the function of list.page?

**Review by brandlxee11 — CHANGES_REQUESTED** [review]
What is the function of list.page at this point in time?

**Comment by bgriffiniv** [comment]
> What is the function of list.page at this point in time?

File: src/app/list/list.page.ts (hunk 2 above)
Comment (brickgriff) [inline src/app/list/list.page.ts:28]: This part assigns a random icon to each item.

File: src/app/list/list.page.ts [hunk 3]
```diff
@@ -28,10 +28,15 @@ export class ListPage implements OnInit {
...
       });
     }
-    console.log("List page started");
+    console.log("List page started (constructor)");
+  }
+
+  setSelectedItem(item) {
+    this.selectedItem = item;
```
Comment (brickgriff) [inline src/app/list/list.page.ts:35]: This just controls which item displays as selected.

File: src/app/profile/profile.page.ts (hunk 1 above)
Comment (brandlxee11) [reply src/app/profile/profile.page.ts]: That's good to know

File: src/app/list/list.page.ts (hunk 2 above)
Comment (brandlxee11) [reply src/app/list/list.page.ts:28]: Cool

**Review by brandlxee11 — APPROVED** [review]
Looks good

**Review by Yogen1337 — APPROVED** [review]
Looks good 👍

**Pull request merged**